nodeJS/src/modules/controllers/task-controller.js

```diff
@@ -21,12 +20,14 @@ module.exports.createNewTask = (req, res, next) => {
 
 module.exports.changeTaskInfo = (req, res, next) => {
   const body = req.body;
-  if ((body.hasOwnProperty('_id')) || (body.hasOwnProperty('text')) || (body.hasOwnProperty('isCheck'))) {
+  if ((body.hasOwnProperty('_id')) && (body.hasOwnProperty('text')) || (body.hasOwnProperty('isCheck'))) {
     tasks.findOneAndUpdate(
       { _id: body._id },
       { $set: body }
     ).then(result => {
-      res.send(result)
+      tasks.find().then(result => {
+        res.send(result);
```

Comment:
There are several problems here:
- why are you setting the result variable twice? Do you understand that these are 2 different parameters? And which one do you send to res.send? And do you need a second one?
- further, why do you need find() if you use findOneAndUpdate()?
- and I asked to be divided into 2 requests. One for changing the checkbox, the other for changing the text.

Reply:
if you use "put" then you can't change 2 parameters.
Do you mean the difference between "put" and "patch"?